management/roles.py

```diff
@@ -93,22 +94,68 @@ class Aura_Teller(Assassin):
 
     def __init__(self):
         self.name = "Aura Teller"
-        self.killers = ["Innocent", "Assassin", "Barber", "Cult Leader", "Executioner", "Huntress", "Witch", "Werewolf", "Lone Wolf", "Devil", "Wager", "Horseman", "Pyromancer"]
 
-    #TODO: Add a function that returns one's aura.
-    def power(self,me):
-        pass
+    def power(self,me,victim):
+        if me.undead == True:
+            return Mailbox().respond("I'm sorry, man! Aura Tellers lose their powers once they turn undead!",me.channel)
+        if me.uses > 0:
+            if me.id == victim.id:
+                return Mailbox().respond("I'm sorry, buddy! Suicide is not an option. Choose again!",me.channel)
+
+            for player in Game_Control().participants:
+                if victim.id == player.id and player.role.name not in ["Spectator", "Dead"]:
+                    me.uses += -1
+                    if player.role.name in ["Werewolf", "Bloody Butcher", "Hell Hound", "Infected Wolf", "Sacred Wolf", "White Werewolf", "Wolf's Cub"]:
+                        mail = Mailbox().respond("{} - <@{}> appears to you as a **Threat**!".format(player.emoji,player.id),me.channel)
+                        mail.respond("This means they're part of the wolf pack!",me.channel)
+                        return mail.log("The **Aura Teller** <@{}> has inspected <@{}>, a **{}**, and discovered them as a threat!".format(me.id,player.id,player.role.name))
+                    Mailbox().respond("{} - <@{}> does **not** appear to you as a threat.".format(player.emoji,player.id),me.channel)
+                    return mail.log("The **Aura Teller** <@{0}> has inspected <@{1}>. As <@{1}> is a **{2}**, they did not appear as a threat to them.".format(me.id,player.id,player.role.name))
+            return Mailbox().respond("Sorry, who could tell that I couldn't find your target?",me.channel)
+        return Mailbox().respond("Sorry, buddy. You can't use your powers right now.",me.channel)
 
 # ===============================================
 class Baker(Innocent):
     
     def __init__(self):
         self.name = "Baker"
-        self.killers = ["Innocent", "Assassin", "Barber", "Cult Leader", "Executioner", "Huntress", "Witch", "Werewolf", "Lone Wolf", "Devil", "Wager", "Horseman", "Pyromancer"]
 
 # ===============================================
 class Butcher(Innocent):
     
     def __init__(self):
         self.name = "Butcher"
-        self.killers = ["Innocent", "Assassin", "Barber", "Cult Leader", "Executioner", "Huntress", "Witch", "Werewolf", "Lone Wolf", "Devil", "Wager", "Horseman", "Pyromancer"]
+
+# ===============================================
+class Barber(Innocent):
+
+    def __init__(self):
+        self.name == "Barber"
+    
+    def night(self,me):
+        me.uses = 0
```

Comment:
This is going to reset a barber's allowed use every time `night()` is triggered. If the barber doesn't use their ability before the first `night()`, then they will lose their killing power. Intentional?

Reply:
Intentional. The goal is to give their ability back with day(), thus allowing them to use their power only during the day.
We can give their use back EVERY day, as we know that the player turns innocent after their first try; we KNOW they won't have used it yet if we get to change it.